code/game/machinery/airlock_control.dm

```diff
@@ -18,7 +18,7 @@
 	update_appearance()
 
 	stoplag(0.2 SECONDS)
-	open(forced = TRUE)
```

Comment:
this line is what sent me into a rage btw. it's not the fault of whoever put this in it's the fault of the framework being non-understandable at-a-glance